torchbenchmark/models/lit_llama/__init__.py

```diff
@@ -0,0 +1,54 @@
+from ...util.model import BenchmarkModel
+from torchbenchmark.tasks import NLP
+import torch
+import os
+from torchbenchmark import REPO_PATH
+import sys
+import lightning as L
+
+LIT_LLAMA_PATH = os.path.join(REPO_PATH, "submodules", "lit-llama")
+
+sys.path.insert(0, LIT_LLAMA_PATH)
```

Comment:
Is it better to use the `add_path()` context manager here (https://github.com/pytorch/benchmark/blob/ae3534ab071270d2d3bb02305c2a1d5089eeb169/torchbenchmark/__init__.py#L27)? If people are running 2 models in the same process, this might cause unexpected results.

Reply:
okey. Note that there are a bunch of other models doing it the wrong way:

```
torchbenchmark/canary_models/fambench_dlrm/__init__.py:        sys.path.insert(0, self.path)
torchbenchmark/models/attention_is_all_you_need_pytorch/preprocess.py:        sys.path.insert(0, path)
torchbenchmark/models/fambench_xlmr/__init__.py:        sys.path.insert(0, self.path)
torchbenchmark/models/pytorch_CycleGAN_and_pix2pix/scripts/edges/batch_hed.py:sys.path.insert(0, caffe_root + 'python')
torchbenchmark/models/pytorch_CycleGAN_and_pix2pix/scripts/eval_cityscapes/cityscapes.py:        sys.path.insert(0, '{}/scripts/helpers/'.format(self.dir))
torchbenchmark/models/tacotron2/waveglow/mel2samp.py:sys.path.insert(0, 'tacotron2')
```

Reply:
For fambench_dlrm and fambench_xlmr, they are part of the `add_path` context manager:  https://github.com/pytorch/benchmark/blob/main/torchbenchmark/canary_models/fambench_dlrm/__init__.py#L16

For other models (attention_is_all_you_need_pytorch, pytorch_CycleGAN_and_pix2pix, and tacotron2), I believe those code are not executed by the model (for example, they are preprocessing code)